### PR TITLE
Update template projects to work with `near dev-deploy`

### PR DESCRIPTION
**Pull request by vgrichina**

### CODE REVIEW

File: blank_project/src/config.js [hunk 1]
```diff
@@ -1,64 +1,65 @@
-(function() {
-    const CONTRACT_NAME = 'nearapp'; /* TODO: fill this in! */
-    const DEFAULT_ENV = 'development';
+const CONTRACT_NAME = process.env.CONTRACT_NAME ||'nearapp'; /* TODO: fill this in! */
```
Comment (mikedotexe) [inline blank_project/src/config.js]: I'm a very big fan of trying an environment variable first. 👍

File: blank_react_project/src/config.js [hunk 2]
```diff
@@ -1,67 +1,65 @@
...
-                nodeUrl: 'http://staging-shared-test.nearprotocol.com:3030',
-                contractName: CONTRACT_NAME,
-                masterAccount: 'test.near',
-            };
-        default:
-            throw Error(`Unconfigured environment '${env}'. Can be configured in src/config.js.`);
-        }
-    }
-    
-    let Cookies = require('js-cookie');
```
Comment (mikedotexe) [inline blank_react_project/src/config.js]: excellent. removing this `require` will fix the issue we discussed before, right @vgrichina? (The one where we used repl to see the true error of why near-shell was acting strange.

**Comment by mikedotexe** [comment]
Seeing this on the `blank_react_project`

```
log.js:24 [HMR] Waiting for update signal from WDS...
index.js:10 nearConfig {networkId: "default", nodeUrl: "https://rpc.nearprotocol.com", contractName: "mikedotdmg", walletUrl: "https://wallet.nearprotocol.com", helperUrl: "https://near-contract-helper.onrender.com"}
App.js:30 come in sign in flow
json-rpc-provider.js:36 Uncaught (in promise) Error: Querying call/mikedotdmg/welcome failed: wasm execution failed with error: FunctionCallError(HostError(GuestPanic("Value with Key: name is not a string or null, filename: \"nearFile\" line: 345 col: 2"))).
{
  "block_height": 869289,
  "error": "wasm execution failed with error: FunctionCallError(HostError(GuestPanic(\"Value with Key: name is not a string or null, filename: \\\"nearFile\\\" line: 345 col: 2\")))",
  "logs": []
}
```

Peter just mentioned I may want to make sure nearlib is up to date in the dependencies. Will look at this after lunch.

**Comment by mikedotexe** [comment]
> Seeing this on the `blank_react_project`
> 
> ```
> log.js:24 [HMR] Waiting for update signal from WDS...
> index.js:10 nearConfig {networkId: "default", nodeUrl: "https://rpc.nearprotocol.com", contractName: "mikedotdmg", walletUrl: "https://wallet.nearprotocol.com", helperUrl: "https://near-contract-helper.onrender.com"}
> App.js:30 come in sign in flow
> json-rpc-provider.js:36 Uncaught (in promise) Error: Querying call/mikedotdmg/welcome failed: wasm execution failed with error: FunctionCallError(HostError(GuestPanic("Value with Key: name is not a string or null, filename: \"nearFile\" line: 345 col: 2"))).
> {
>   "block_height": 869289,
>   "error": "wasm execution failed with error: FunctionCallError(HostError(GuestPanic(\"Value with Key: name is not a string or null, filename: \\\"nearFile\\\" line: 345 col: 2\")))",
>   "logs": []
> }
> ```
> 
> Peter just mentioned I may want to make sure nearlib is up to date in the dependencies. Will look at this after lunch.

Not a problem, actually. I approve this PR, good stuff.

**Comment by vgrichina** [comment]
@mikedotexe `blank_react_project` not yet updated, sorry for confusion

File: blank_react_project/src/config.js (hunk 2 above)
Comment (vgrichina) [reply blank_react_project/src/config.js]: yes

**Pull request merged**